### DISCUSSION TITLE
embeded iframe from code sand box is giving lots of warming and few errors?

### DISCUSSION THREAD

**Discussion by Tula-Magar** [codesandbox/codesandbox-client#6713] [Q&A]
Here is the link where I explained and posted:
https://stackoverflow.com/questions/72385638/embeded-iframe-from-code-sand-box-is-giving-lots-of-warming-and-few-errors

**Comment by JamesACS** [comment]
Hey @Tula-Magar - I'm afraid there isn't much we can do with that report. Do you have a link to the Sandbox in question, as well as any reproduction steps?
